### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by sowu880**
Add template management in readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,8 +60,26 @@ Here are the parameters that the tool accepts:
 
 Example usage to convert HL7 v2 messages to FHIR resources in a folder:
 ```
->.\Microsoft.Health.Fhir.Liquid.Converter.Tool.exe -d myTemplateDirectory -e ADT_A01 -i myInputDataFolder -o myOutputDataFolder
+>.\Microsoft.Health.Fhir.Liquid.Converter.Tool.exe convert -d myTemplateDirectory -e ADT_A01 -i myInputDataFolder -o myOutputDataFolder
```
Comment (ranvijaykumar) [inline README.md:63]: -e should be replaced by -r

File: README.md [hunk 2]
```diff
@@ -58,8 +60,26 @@ Here are the parameters that the tool accepts:
...
+```
+
+**Manage Templates**
```
Comment (ranvijaykumar) [inline README.md:66]: Let's move the 'manage template' under the '$convert-data operation in the Azure API for FHIR' section with appropriate context. This information is relevant there.

File: README.md [hunk 3]
```diff
@@ -58,8 +60,26 @@ Here are the parameters that the tool accepts:
...
+
+The command-line tool also supports managing different versions of templates from Azure Container Registry (ACR). Users can customize templates and store them on ACR if default templates can not meet requirements. After ACR authentication, users can pull and push templates from/to a remote ACR through our tool.
```
Comment (ranvijaykumar) [inline README.md:68]: Provide a link, or note on how to do ACR authentication.

File: README.md [hunk 4]
```diff
@@ -58,8 +60,26 @@ Here are the parameters that the tool accepts:
...
 ```
+Example usage of pulling an image of templates in a folder:
+
+```
+>.\Microsoft.Health.Fhir.Liquid.Converter.Tool.exe pull testacr.azurecr.io/templatetest@sha256:412ea84f1bb1a9d98345efb7b427ba89616ec29ac332d543eff9a2161ca12a58 myOutputFolder
+
+```
+More details of usage are given in [Template Management CLI tool](docs/TemplateManagementCLI.md).
+
+Besides current version of [templates](data/Templates) given in our project, other versions that released by Microsoft are stored in a public ACR: healthplatformregistry.azurecr.io, users only have pull permission and can directly pull templates without authentication.
```
Comment (ranvijaykumar) [inline README.md:82]: How can the users list/see those templates?

Comment (ranvijaykumar) [reply README.md:82]: See previous comments

**Review by ranvijaykumar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sowu880) [reply README.md]: Fixed it, thanks.

File: README.md (hunk 2 above)
Comment (sowu880) [reply README.md:66]: This part is the usage of CLI for template management. Seems in 'command line tool' section is more property?

Comment (ranvijaykumar) [reply README.md:66]: I see; we can keep it here.

File: README.md (hunk 4 above)
Comment (sowu880) [reply README.md]: For now, we don't have this feature, but the versions of templates are aligned with converter engine that users can easily find.

**Review by ranvijaykumar — APPROVED** [review]

**Pull request merged**